Tasks/WindowsMachineFileCopyV2/Utility.ps1

```diff
@@ -54,4 +54,18 @@ function Copy-OnLocalMachine(
 {
     $credential = New-Object 'System.Net.NetworkCredential' -ArgumentList $adminUserName, $adminPassword
     Invoke-Command -ScriptBlock $CopyJob -ArgumentList "", $sourcePath, $targetPath, $credential, $cleanTargetBeforeCopy, $additionalArguments
+}
+
+function Try-CleanupPSDrive (
+	[string] $path
+	)
+{
+	try {
+		Write-Verbose "Executing cmd.exe /c net use /delete `"$path`" `"2>NUL`""
+		cmd.exe /c net use /delete "$path" "2>NUL"
+	}
+	catch {
+		#Ignore the error if any
+		Write-Verbose "Unable to remove path: $path. Ignoring error message: $($_.Exception.Message)"
+	}
```

Comment:
nit: You could also add the CmdletBinding decorator on the function. Then the caller could just do `Try-CleanupPSDrive -Path $remoteSharePath -ErrorAction SilentlyContinue`